res/values-in/strings.xml

```diff
@@ -93,9 +93,9 @@
     <string name="Level">"Level"</string>
     <string name="PAUSED">"DIJEDA"</string>
     <string name="FRIENDS">"TEMAN"</string>
-    <string name="Warning">"Peringatan"</string>
+    <string name="Warning">"Peringatan!"</string>
```

Comment:
No any exclamation needed here. Keep in mind that translations must be based at English file (`res/values`).